great_expectations/compatibility/sqlalchemy_and_pandas.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import warnings
+from typing import Callable
+
+import pandas as pd
+
+from great_expectations.optional_imports import (
+    SQLALCHEMY_NOT_IMPORTED,
+    is_version_greater_or_equal,
+    is_version_less_than_or_equal,
+    sqlalchemy_version_check,
+)
+from great_expectations.warnings import (
+    warn_pandas_less_than_1_4_and_sqlalchemy_greater_than_or_equal_2_0,
+)
+
+try:
+    import sqlalchemy as sa
```

Comment:
Can we import `sqlalchemy` from `great_expectations.optional_imports`?

Reply:
🤦 Yes of course - good catch!